packages/core/src/awsService/appBuilder/serverlessLand.ts

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+import * as path from 'path'
+// import * as vscode from 'vscode'
+import { getTelemetryResult, RegionProvider } from '../../shared'
```

Comment:
You can find other instances of this import to see how this is handled in other places (to not have the error here below) (in this case, is to import from `../../shared/errors` instead of just `../../shared`)

Reply:
Corrected it. Thanks!